extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Anchor.fm integration.
+ *
+ * @since 9.0.0
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\AnchorFm;
+
+use Automattic\Jetpack\Blocks;
+use Jetpack_Podcast_Helper;
+
+const FEATURE_NAME = 'anchor-fm';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
```

Comment:
Since it's not autoloaded, I think we may have to double check and require it at all times:

```suggestion

if ( ! class_exists( 'Jetpack_Podcast_Helper' ) ) {
	\jetpack_require_lib( 'class-jetpack-podcast-helper' );
}

```